content/en/_index.md

```diff
@@ -9,7 +9,10 @@ title: Parquet
 <a class="btn btn-lg btn-secondary me-3 mb-4" href="/blog/">
   Download <i class="fab fa-github ms-2 "></i>
 </a>
-<p class="lead mt-5">Apache Parquet is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.</p>
+<p class="lead mt-5">
```

Comment:
I wordsmithed the landing page a little to reduce its length and make it flow better. I can make it exactly mirror the DOAP text if reviewers prefer

Reply:
At the risk of bike-shedding this (+1 as-is), I think it might be good to indicate that Parquet has readers/writers in many languages and in many tools. In the recent [IANA registration](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) I went with the vague "used across a wide variety of platforms, technologies, and environments.". But here maybe,

> "Parquet is supported in many programming language and analytics tools."

What do you think?

Reply:
I think the generic sentiment of wide support is good 👍 

I tried to remove the specific Java/C++/Go tech references which are on https://projects.apache.org/project.html?parquet (comes from https://github.com/apache/parquet-site/blob/production/static/doap_Parquet.rdf it seems)

What do you think about changing all three locations (DOAP, this, and the overview @etseidl  mentions in https://github.com/apache/parquet-site/pull/59/files#r1599056646) to use the more generic phrasing?


> "Parquet is supported in many programming language and analytics tools."

Reply:
SGTM

Reply:
My plan is to wait another day or so for any additional comments, and if I don't hear any I will update this PR so that all three locations use the generic "Parquet is supported in many programming language and analytics tools." phrasing

Reply:
Text is updated